rio_tiler/sentinel2.py

```diff
@@ -14,64 +16,81 @@
 from rasterio.warp import transform_bounds
 
 from rio_tiler import utils
-from rio_tiler.errors import TileOutsideBounds, InvalidBandName, InvalidSentinelSceneId
-
-SENTINEL_BUCKET = "s3://sentinel-s2-l1c"
-SENTINEL_BANDS = [
-    "01",
-    "02",
-    "03",
-    "04",
-    "05",
-    "06",
-    "07",
-    "08",
-    "8A",
-    "09",
-    "10",
-    "11",
-    "12",
-]
+from rio_tiler.errors import (
+    DeprecationWarning,
+    TileOutsideBounds,
+    InvalidBandName,
+    InvalidSentinelSceneId,
+)
+
+AWS_SENTINEL_BUCKET = "s3://sentinel-s2-"
+
+SENTINEL_L1_BANDS = {
+    "10m": ["02", "03", "04", "08"],
+    "20m": ["05", "06", "07", "11", "12", "8A"],
+    "60m": ["01", "09", "10"],
+}
+
+SENTINEL_L2_BANDS = {
+    "10m": ["02", "03", "04", "08"],
+    "20m": ["02", "03", "04", "05", "06", "07", "08", "11", "12", "8A"],
+    "60m": ["01", "02", "03", "04", "05", "06", "07", "08", "09", "11", "12", "8A"],
+}
+
+SENTINEL_L2_PRODUCTS = {
+    "10m": ["AOT", "WVP"],
+    "20m": ["AOT", "SCL", "WVP"],
+    "60m": ["AOT", "SCL", "WVP"],
```

Comment:
@vincentsarago I'm assuming the reason `TCI` isn't exposed here is because it's a 3-band dataset, unlike all the other single-band options? I suppose that would throw a wrench into the current rio-tiler strategy.

It's a bit unfortunate that TCI isn't trivial to support. I saw in your recent [COG Talk 4ter - Distributed processes](https://medium.com/devseed/cog-talk-4ter-distributed-processes-8ee280f71080) that you're specifying rio-color params to get bands 04,03,02 to look good. I use a similar strategy but my results are usually significantly less pleasing than these TCI images 😄.

I can imagine someday adding an additional arg to `sentinel2.tile()` that could allow the user to request the TCI imagery. Or even splitting TCI into three individual bands that could be specified in the bands arg `bands=("TCI1", "TCI2", "TCI3")`

Reply:
yes this is intentional because you can easily read the TCI using `rio_tiler.main.tile` function. 

```
from rio_tiler.sentinel2 import _sentinel_parse_scene_id
from rio_tiler.main import tile

tci_path = os.path.join(
        scene_params["aws_bucket"],
        scene_params["aws_prefix"],
        "R10m",
        "TCI.jp2"
)
t, m = tile(tci_path, x, y, z)
```

Basically, if we were adding multiple bands file support (e.g TCI) in addition to single band it will be a nightmare to handle.

Reply:
> if we were adding multiple bands file support (e.g TCI) in addition to single band it will be a nightmare to handle

Yes, totally, I understand completely.

Thanks for the suggestion. That's a great strategy.